docs/internals/security.txt

```diff
@@ -38,6 +38,32 @@ action to be taken, you may receive further followup emails.
 
 .. _our public Trac instance: https://code.djangoproject.com/query
 
+.. _reporting-guidelines:
+
+Reporting guidelines
+====================
+
+An identified vulnerability should meet the following guidelines to be
+considered valid:
```

Comment:
Do you have a wording suggestion (there's also the section above)?

Reply:
Perhaps something along the lines of:

```suggestion
considered valid:

If you are unsure whether your finding meets these guidelines, it's preferred to proceed with caution and report it :ref:`properly
  <reporting-security-issues>` as opposed to publicly. The security team will review your report and recommend the correct course of action.
```

(Probably as a note at the bottom of the section, rather than here). There might also be some value in moving this note up, since it's also true for a "security-looking bug".

Reply:
Added a note to the bottom :+1: